### PR TITLE
Add TimeStatsAsync to MergeRequestClient

### LINKED ISSUE #538: Missing TimeStatsAsync for MergeRequests
Hi everyone and thank you for this awesome library!

Currently it is possible to access time stats on issues:

```csharp
var timestats = await _gitLabClient.Issues.TimeStatsAsync(
  issue.ProjectId,
  issue.IssueId,
  CancellationToken.None);
```

The same does not work with merge requests as the `TimeStatsAsync` is missing:

```csharp
var timestats = await _gitLabClient.MergeRequests.TimeStatsAsync(
  mergeRequest.ProjectId,
  mergeRequest.Id,
  CancellationToken.None);
```

This API does not seem to exist. According to [GitLab documentation](https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests) the time stats are provided along with merge requests themselves.

Can the merge request model be extended with a `TimeStats` property or the client be extended with `TimeStatsAsync` method?

### PR DESCRIPTION
**Pull request by DelphinRP**
Adds `TimeStatsAsync` to `MergeRequestClient` allowing fetching time stats for merge requests via

```csharp
var timestats = await _gitLabClient.MergeRequests.TimeStatsAsync(mergeRequest.ProjectId, mergeRequest.Iid, CancellationToken.None);
```

Closes #538

### CODE REVIEW

File: NGitLab/Impl/MergeRequestClient.cs [hunk 1]
```diff
@@ -156,6 +158,11 @@ public GitLabCollectionResponse<MergeRequestVersion> GetVersionsAsync(int mergeR
             return _api.Get().GetAllAsync<MergeRequestVersion>(_projectPath + "/merge_requests/" + mergeRequestIid.ToString(CultureInfo.InvariantCulture) + "/versions");
         }
 
+        public Task<TimeStats> TimeStatsAsync(int projectId, int mergeRequestIid, CancellationToken cancellationToken = default)
```
Comment (Toa741) [inline NGitLab/Impl/MergeRequestClient.cs]: It seems that all other methods in MergeRequestClient already knows the projectId.

Could you remove the param and use the existing `_projectPath` ?

Comment (DelphinRP) [reply NGitLab/Impl/MergeRequestClient.cs]: Done

**Comment by DelphinRP** [comment]
@Toa741 The branch getting outdated with commits to the main branch prevents auto merge from triggering.

**Comment by DelphinRP** [comment]
@Toa741 @phdesroUbi @meziantou  Is there anything we can do to speed up the merge of this pull request? Review is done, but the auto merge is not going through...

**Comment by Toa741** [comment]
> @Toa741 @phdesroUbi @meziantou Is there anything we can do to speed up the merge of this pull request? Review is done, but the auto merge is not going through...

@DelphinRP, my bad, sorry for the delay

**Pull request merged**